app/src/main/res/layout/activity_article_details.xml

```diff
@@ -1,87 +1,118 @@
     android:layout_height="fill_parent">
     <!-- Here you put the rest of your current view-->
 <androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:padding="8dp">
 
-    <TextView
-        android:layout_width="match_parent"
+    <WebView
+        android:id="@+id/webview"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        app:layout_constraintBottom_toBottomOf="@+id/article_image1"
-        />
-
-
-    <ImageView
-        android:id="@+id/article_image1"
-        android:layout_width="match_parent"
-        android:layout_height="200dp"
-        android:scaleType="centerCrop"
-        app:layout_constraintTop_toTopOf="parent"/>
-
+        android:layout_marginTop="80dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/scrollView2" />
 
     <TextView
         android:id="@+id/article_title"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:textSize="20sp"
+        android:layout_marginStart="4dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="4dp"
+        android:text="title of the news"
+        android:textColor="@color/darkblue"
+        android:textSize="16sp"
         android:textStyle="bold"
-        android:padding="16dp"
-        app:layout_constraintTop_toBottomOf="@id/article_image1"/>
-
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
-        android:id="@+id/article_description"
-        android:layout_width="match_parent"
+        android:id="@+id/article_published_date1"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:textSize="16sp"
-        android:padding="16dp"
-        app:layout_constraintTop_toBottomOf="@id/article_title"/>
+        android:layout_marginTop="12dp"
+        android:text="24 june 2019"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.012"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/article_title" />
 
-    <TextView
-        android:id="@+id/article_source"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:textSize="14sp"
-        android:textColor="@android:color/darker_gray"
-        android:padding="16dp"
-        app:layout_constraintTop_toBottomOf="@id/article_description"
-        />
+    <ImageView
```

Comment:
Remove these unnecessary views. Juat load the news into the web view. You can consider putting the web view as a parent. And also please reseign the page with views that make sense.